torch_xla/csrc/ops/topk.cpp

```diff
@@ -18,6 +18,23 @@ xla::Shape NodeOutputShape(const torch::lazy::Value& input, int64_t k,
   return InferOutputShape({GetXlaShape(input)}, lower_for_shape_fn);
 }
 
+xla::Shape NodeOutputShapeSymInt(const torch::lazy::Value& input,
+                                 int64_t k_upper_bound, int64_t dim,
+                                 bool largest, bool sorted, bool stable) {
+  xla::Shape input_shape = GetXlaShape(input);
+  std::vector<int64_t> dimensions(input_shape.dimensions().begin(),
+                                  input_shape.dimensions().end());
+  XLA_CHECK_LT(dim, input_shape.rank());
+  dimensions[dim] = k_upper_bound;
+  xla::Shape values_shape =
+      xla::ShapeUtil::MakeShape(input_shape.element_type(), dimensions);
+  xla::Shape indices_shape =
+      xla::ShapeUtil::MakeShape(xla::PrimitiveType::S64, dimensions);
```

Comment:
index shape is actully s32, can you use https://github.com/pytorch/xla/blob/master/torch_xla/csrc/tensor_util.cpp#L1254 ? context is `SetDimensionShape` and `getDimensionShpe` currently only handle s32, this is where we have slight difference with upstream.

Reply:
Actually the static version also returns s64 type, because of a s32 to s64 cast here: https://github.com/pytorch/xla/blob/1bbe4da79753bf0a4bd94ff106cf1267f841f302/torch_xla/csrc/xla_lower_util.cpp#L380-L383
Do I also need to disable this cast?